COPYING.md

```diff
@@ -5,8 +5,7 @@
 
     This program is free software: you can redistribute it and/or modify
     it under the terms of the GNU General Public License as published by
-    the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
+    the Free Software Foundation, version 3.
```

Comment:
This is how SPDX does it: https://spdx.org/licenses/GPL-3.0-only.html (at the end of the page)